webapp-django/crashstats/crashstats/static/crashstats/js/socorro/dashboard_graph.js

```diff
@@ -1,110 +1,111 @@
-/*global window:true, $:true, socorro:true, crashReportsByBuildDateTmpl:true, crashReportsByVersionTmpl:true, Mustache:true */
-$(function() {
-    "use strict";
-    var chartContainer = $("#adu-chart"),
-        product = "",
-        version = "",
-        baseURL = "/home/frontpage_json?",
-        ajaxURL = "",
-        hashString = "",
-        currentHref = "",
-        dateRangeType = "",
-        dateRangeVal = "",
-        dateRangeTypeValPattern = /(?!=)[a-zA-Z]{1,6}(?=:|$)/i,
-        durationPattern = /\d{1,2}(?=:|$)/,
-        durationValPattern = /(?!=)\d{1,6}(?=:|$)/,
-        colours = ['#6a3d9a', '#e31a1c', '#008800', '#1f78b4'];
-
-    var setSelected = function(item, container) {
-        $(container).find('a').removeClass("selected");
-        item.addClass("selected");
-    },
-    setSelectedByHash = function() {
-        hashString = $.trim(window.location.hash);
-        if(hashString.length) {
-            if(dateRangeTypeValPattern.exec(hashString) !== null) {
-                $("#date-range-type a").each(function() {
-                    if($(this).attr("href").indexOf(dateRangeTypeValPattern.exec(hashString)) > -1) {
-                        setSelected($(this), "#date-range-type");
+/*global window:true, $:true, socorro:true, MG: true */
+$(function () {
+    'use strict';
+
+    var chartContainer = $('#adu-chart');
+    var baseURL = '/home/frontpage_json?';
```

Comment:
Still gross that it's a hardcoded URL in the javascript. I'd much rather we pick this up via the DOM or something.